mule-packager/src/main/java/org/mule/tools/api/packager/builder/DomainBundlePackageBuilder.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.tools.api.packager.builder;
+
+import org.codehaus.plexus.archiver.ArchiverException;
+import org.mule.tools.api.packager.archiver.Archiver;
+import org.mule.tools.api.packager.archiver.DomainBundleArchiver;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static com.google.common.base.Preconditions.checkNotNull;
+import static com.google.common.base.Preconditions.checkState;
+import static org.mule.tools.api.packager.structure.FolderNames.*;
+
+public class DomainBundlePackageBuilder implements PackageBuilder {
+
+  private File destinationFile;
+  private DomainBundleArchiver archiver;
+  private File domainFolder = null;
+  private File applicationsFolder = null;
+  private File mavenFolder = null;
+
+
+  public DomainBundleArchiver getDomainBundleArchiver() {
+    if (archiver == null) {
+      archiver = new DomainBundleArchiver();
+    }
+    return archiver;
+  }
+
+  /**
+   * @param file file to be created with the content of the domain bundle
+   * @return
+   */
+  public DomainBundlePackageBuilder withDestinationFile(File file) {
```

Comment:
pelase improve the docs 
it should say something like -> deifines the destination file where to leave the build package
return this builder